llvm/test/CodeGen/AMDGPU/live-interval-bug-in-rename-independent-subregs.mir

```diff
@@ -245,40 +245,41 @@ body: |
   ; DeadInstDelete-NEXT:   liveins: $sgpr4, $sgpr5, $sgpr6, $sgpr7, $sgpr14, $vgpr0, $sgpr8_sgpr9:0x000000000000000F, $sgpr10_sgpr11, $sgpr12_sgpr13, $sgpr0_sgpr1_sgpr2_sgpr3
   ; DeadInstDelete-NEXT: {{  $}}
   ; DeadInstDelete-NEXT:   renamable $sgpr15 = S_OR_B32 renamable $sgpr14, 2, implicit-def dead $scc
-  ; DeadInstDelete-NEXT:   renamable $vgpr8_vgpr9 = IMPLICIT_DEF
-  ; DeadInstDelete-NEXT:   renamable $vgpr1_vgpr2 = IMPLICIT_DEF
-  ; DeadInstDelete-NEXT:   renamable $vgpr0_vgpr1_vgpr2_vgpr3_vgpr4_vgpr5_vgpr6_vgpr7 = IMPLICIT_DEF
+  ; DeadInstDelete-NEXT:   renamable $vgpr15 = COPY killed renamable $sgpr15
```

Comment:
Added run line that shows instruction that was deleted.
Here, after the change, it is not deleted. 
For context this is lowered divergent phi with uniform inputs, inputs stayed in sgpr but copy to vgpr(phi itself) got deleted because it became dead after register allocator that did not have correct live intervals information.